src/main/resources/assets/unicopia/lang/en_us.json

```diff
@@ -817,7 +817,7 @@
   "gui.unicopia.spellbook.chapter.introduction.p3.1.body": "Unusual Rocks",
   "gui.unicopia.spellbook.chapter.introduction.p3.2.body": "These 'Gemstones' as the locals call them are a common material found around the world. Farm-Ponies dig them up all the time and consider it a local delicacy, but I believe these stones are capable of a lot more than they let on.",
   "gui.unicopia.spellbook.chapter.introduction.p4.title": "Gemstones",
-  "gui.unicopia.spellbook.chapter.introduction.p4.1.body": "My research is still incomplete but I may have stumbled upon something. These stones have high magical potentia! More than I've ever seen before!",
+  "gui.unicopia.spellbook.chapter.introduction.p4.1.body": "My research is still incomplete but I may have stumbled upon something. These stones have high magical potential! More than I've ever seen before!",
```

Comment:
Can this one be changed back please? potentia is the intended spelling.

Reply:
https://en.wiktionary.org/wiki/potentia

Reply:
Reverted. Some changes I made only because Intellij told me so.